src/utils/okduris.js

```diff
@@ -1,10 +1,11 @@
 import { get } from 'lodash';
 
 export const getSubPagePath = (apiObj, model, subPage) => {
-  if (!apiObj) {
+  if (!apiObj || !model) {
```

Comment:
How can this happen? The model is statically provided at build-time. I guess it's a bug if it is missing here.

Reply:
I guess it would be a bug if it wasn't there. I was just looking at it as a block and adding more validation to this function.